CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 ### Added
 
+- `sandstone/Input` and `sandstone/Input.InputPopup` url to prop `type`
```

Comment:
```suggestion
- `sandstone/Input` `url` to prop 'type'
```

Reply:
Did you mean `InputField` instead of `InputPopup`?
Technically, `InputField` has string `type` so we can't say "added `url`" to the prop.
I guess we just updated the samples and tests. =)

Reply:
🤔 Oh, we do have `InputPopup` component.. 😃

Reply:
I mean InputPopup in this issue as I added `url` to the list of acceptable values for `type` prop in `InputBase` and `InputPopupBase`.
As you mentioned, InputField has type `string` so I didn't fixed `InputField` this time.
However, InputField samples and tests already have cases of type `number` and `password`, so I added some samples and test with `InputField`.

Reply:
🙆

Reply:
It's fixed.